AcceptanceTest.md

```diff
@@ -0,0 +1,23 @@
+# AcceptanceTest for QualityToolsExtension.jar
```

Comment:
Is there **exactly one and only one** acceptance test?  Is a new file needed for an additional test?

Is the intent that this only apply to a file named _QualityToolsExtension.jar_?  Does it need to be rewritten or replaced if the name of the jar file evolves during the lifetime of the product?

Reply:
1) Within the ```AcceptanceTest.md``` file.  There are two acceptance tests.  First test is to verify ```QualityTools``` Extension displays.  The second acceptance test being Quality Tools Extension displays from Preferences “**Extensions Tab**”.

2) It seems our organization has decided to implement that name for our production jar files as in User story #149 and #138 which could be confusing. We are proposing that we name it ```ProductAcceptanceTest.md```

Reply:
It seems the MetroCS organization plans on the future implementation of semantic versioning in our production jar files as detailed in User story #149 and #138. However, slight modifications will need to be updated ``` "rewritten or replaced" ``` in the file name and body content until a more permanent strategy can be developed to automagically handle semantic versioning of file names. https://semver.org/